components/properties/src/props.rs

```diff
@@ -49,31 +49,31 @@ pub enum EnumeratedProperty {
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[repr(transparent)]
 #[zerovec::make_ule(BidiClassULE)]
-pub struct BidiClass(pub u16);
+pub struct BidiClass(pub u8);
```

Comment:
suggestion: make doc descriptive

Please add a brief description here of what the Unicode property `Bidi_Class` that this enum `BidiClass` represents. Look at the other newtype/enum definitions, it's usually 1-2 lines. You can pull from the definitions in TR44.  Since the API `bidi_class()` says "see `BidiClass` for more info", it's important to actually have something to say here.

Reply:
done